webpack/stories/docs/APIPollingMiddleware.md

```diff
@@ -0,0 +1,77 @@
+# API Middleware with polling
+
+current generic API action looks like this:
+
+```js
+const some_API_action = payload => ({
+  type: API_OPERATIONS.GET,
+  key: MY_API_REQUEST_SPECIAL_KEY,
```

Comment:
If I understand right, this is a key you must generate and supply yourself?  I think it would be good to add a comment here to clarify